specification/schemaregistry/data-plane/Microsoft.SchemaRegistry/stable/2023-07-01/schemaregistry.json

```diff
@@ -0,0 +1,795 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Schema Registry",
+    "version": "2023-07-01",
+    "description": "SchemaRegistryClient is a client for registering and retrieving schemas from the Azure Schema Registry service.",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "https://{fullyQualifiedNamespace}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "name": "fullyQualifiedNamespace",
+        "in": "path",
+        "description": "The Schema Registry service endpoint, for example 'my-namespace.servicebus.windows.net'.",
+        "required": true,
+        "type": "string",
+        "format": "uri",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "OAuth2Auth": [
+        "https://eventhubs.azure.net/.default"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "OAuth2Auth": {
+      "type": "oauth2",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "https://eventhubs.azure.net/.default": ""
+      }
+    }
+  },
+  "tags": [],
+  "paths": {
+    "/$schemaGroups": {
+      "get": {
+        "operationId": "ListSchemaGroups",
```

Comment:
Something to pay attention to, the changes in operation id will probably have an impact on rest api docs